In-house patch that allows tests to pass with Python 3.x.

--- suds-jurko-0.6/tests/test_suds.py.orig	2016-09-02 06:32:25.212848597 -0800
+++ suds-jurko-0.6/tests/test_suds.py	2016-09-02 06:33:29.463753721 -0800
@@ -1921,7 +1921,7 @@ xmlns:soap="http://schemas.xmlsoap.org/w
     assert binding_qname == ("dummy", "my-namespace")
     assert binding.__class__ is suds.wsdl.Binding
     assert len(binding.operations) == 1
-    operation = binding.operations.values()[0]
+    operation = list(binding.operations.values())[0]
     input = operation.soap.input.body
     output = operation.soap.output.body
     assert len(input.parts) == 1
@@ -2060,7 +2060,7 @@ def _element_node_xml(name, min=None, ma
 
 def _first_from_dict(d):
     """Returns the first name/value pair from a dictionary or None if empty."""
-    for x in d.items():
+    for x in list(d.items()):
         return x[0], x[1]
 
 
